_posts/2018-11-16-stairway-to-health.md

```diff
@@ -0,0 +1,94 @@
+When the user hovers over the chart he can also see the evolution of people taking the stair / elevator within that day / week / month / year.
+Next to the chart, the total absolute number of all the observations measured is reported per period.
+
+![PartTwo](/img/2018-11-16-stairway-to-health/PartTwo.gif)
+
+## Part three: the more detailed area chart
+
+The third part visualises how the division stair / elevator is evolving over time expressed in day / week / month or year basis.
+This gives the user the possibility to look at trends and to see how the situation of today compares itself to past situations.
+
+In the title the user has the option to change the appearance of the data (absolute or shares).
+He has also the option to change how many periods are visualised starting from the most recent period.
+
+![PartThreeOne](/img/2018-11-16-stairway-to-health/PartThreeOne.gif)
+
+When the user hovers over the chart he can see the same horizontal bar comparison allowing him to compare the selected period above with the period he is hovering over in the chart.
+
+![PartThreeTwo](/img/2018-11-16-stairway-to-health/PartThreeTwo.gif)
+
+Last feature to discuss here is how the user can change which period the chart is showing.
+This can be done by clicking the chart above.
+When you click on the day bar on the top chart the bottom chart is expressed on a day level.
+This applies also to the other period bases in the chart.
+
+![PartThreeThree](/img/2018-11-16-stairway-to-health/PartThreeThree.gif)
+
+# What can we do next?
+
+While this dashboard already gives an answer to the most important questions and gives the user the possibility to explore the data over time, there are still some extra things that can be developed.
+
+The dashboard is currently built within a Tableau workbook which is using the data of the [MongoDb][] database JWorks set up as an extract.
+This means we don't have a live connection to the actual database as JWorks has in their app.
+This brings us to the first thing we can still explore: **deployment**.
+In order to integrate the dashboard in the original application, we could publish the dashboard on the [Tableau server][] of Ordina which is running on [Microsoft Azure][].
+Running this instance is not free so when taking a decision we should also take the user relevance in consideration: does the user really need to have a live connection to the data or does a nightly update accomplish his needs?
+
+Secondly we can still do a lot on the **analysis part**. What are the reasons why some patterns in the data exist? Do people take the stairs less when it is hot outside?
+JWorks recently tracked on which floor the observation is measured allowing us to look into difference by floor. Do people take the elevator more when they need to go from floor 1 to floor 3?
+
+We will keep you posted about further progress on the things described above. Thank you for reading and don't forget: always take the stairs!
```

Comment:
Proposed rephrase:

We will keep you posted on further progress related to Stairway to Health.